src/features/claim/components/CreateWallet.tsx

```diff
@@ -14,7 +14,10 @@ interface CreateWalletProps {
   redirectUrl?: string;
 }
 
-const defaultWallet = WALLET_OPTIONS[0];
+const { supportedWallets } = getConfig();
+
+const defaultWallet =
```

Comment:
Do you think we should have this in the config instead in case multiple pages need to reference it

Reply:
Sure that's a good idea